### PR TITLE
[Outlook] (closeAsync) Map Script Lab snippet to reference docs

### PR DESCRIPTION
**Pull request by samantharamon**

### CODE REVIEW

File: samples/outlook/99-preview-apis/close-async.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 order: 3
 id: outlook-close-async
 name: Close the current message and discard changes (Message Compose)
-description: Close the current message and discard the changes when specified.
+description: Close the current message and discard any unsaved changes when specified.
```
Comment (ElizabethSamuel-MSFT) [inline samples/outlook/99-preview-apis/close-async.yaml:4]:
```suggestion
description: Closes the current message and discards any unsaved changes when specified.
```

File: samples/outlook/99-preview-apis/close-async.yaml [hunk 2]
```diff
@@ -10,8 +10,9 @@ script:
         $("#close-async").click(closeAsync);
 
         function closeAsync() {
-          // This snippet closes and discards the current message being composed and discards any unsaved changes. The API call works on new messages being composed, inline replies, and existing drafts when the optional property, discardItem, is set to true. When discardItem is set to false or isn't defined and the closeAsync call attempts to close a new message with unsaved changes, the user is prompted to save a draft, discard the changes. or cancel the close operation.
-
+          // This snippet closes the current message being composed and discards any unsaved changes when the optional property, discardItem, is set to true.
+          // The API call works on a new message being composed, reply, or an existing draft.
```
Comment (ElizabethSamuel-MSFT) [inline samples/outlook/99-preview-apis/close-async.yaml:14]:
```suggestion
          // The API call works on a new message being composed, a reply, or an existing draft.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
If you accept the suggestions, remember to rerun yarn start

File: samples/outlook/99-preview-apis/close-async.yaml (hunk 1 above)
Comment (samantharamon) [reply samples/outlook/99-preview-apis/close-async.yaml:4]: Thanks for the suggestion. I'll retain the current wording to keep it consistent with other snippets.

**Pull request merged**